atomicapp/providers/openshift.py

```diff
@@ -461,7 +461,11 @@ def undeploy(self):
             if self.dryrun:
                 logger.info("DRY-RUN: DELETE %s", url)
             else:
-                self.oc.delete(url)
+                try:
+                    self.oc.delete(url)
+                except Exception as e:
+                    if not self.ignore_errors:
+                        raise e
```

Comment:
I think this may be the only part @rtnpro you should have a look at, other code LGTM.